### PR TITLE
added dev attribute to interface source

### PR DESCRIPTION
**Pull request by danielkucera**
Signed-off-by: Daniel Kucera <kucera@apptocloud.com>

### CODE REVIEW

File: domain.go [hunk 1]
```diff
@@ -312,6 +312,7 @@ type DomainGraphicListener struct {
 }
 
 type DomainGraphic struct {
+	XMLName       xml.Name                `xml:"graphics"`
```
Comment (berrange) [inline domain.go]: Why do you think this is needed ? It is only something that needs to be added for structs which form the root of an XML document - DomainGraphic is only ever needed as a child of another element.

**Comment by berrange** [comment]
BTW, i merged your first 3 commits already - just the Q for the last one

File: domain.go (hunk 1 above)
Comment (danielkucera) [inline domain.go]: I am using it as separate document when changing graphics config via domain.UpdateDeviceFlags(dev_xml, libvirt.DOMAIN_DEVICE_MODIFY_CURRENT) and without this patch it marshals into <DomainGraphic> tags.

Comment (berrange) [inline domain.go]: Ah, ok, I didn't realize we actually had support for that wired up.

**Comment by berrange** [comment]
All patches merged now.

**Pull request closed without merging**